### DISCUSSION TITLE
QT Designer Theme

### DISCUSSION THREAD

**Discussion by itsgoodforyou2** [KhamisiKibet/QT-PyQt-PySide-Custom-Widgets#81] [Q&A]
Hello, I am studying various ways to make Python UI through your YouTube. One question is why the theme color of the Qt Designer you use and the Qt Designer I use is different.  For example, what I'm using is a bright color, and what you're using is a dark color.  Especially, the main window color I use is gray, but the one you use is dark, and the finished UI is also made nice with dark colors.  How can I produce UI with dark colors like you?

![image](https://github.com/KhamisiKibet/QT-PyQt-PySide-Custom-Widgets/assets/83760365/97fc88c5-143c-4e60-a4ba-480eedd562f2)

**Comment by spyhabs** [comment]
Change the linux - ubuntu settings in dark mode.
